Makefile

```diff
@@ -2,10 +2,10 @@ all: check-deps prod
 
 debug: debugjs debugcss build/index.debug.html build/not-found.html fonts build/js/compat.js build/js/jquery.min.js build/js/bootstrap.min.js build/sitemap.xml build/strings/locales.json build/index.$(DEFAULT_LOCALE).html build/strings/$(DEFAULT_LOCALE).json images
 
-prod: js css html fonts build/sitemap.xml build/manifest.json build/strings/locales.json localhtml images
+prod: js flow css html fonts build/sitemap.xml build/manifest.json build/strings/locales.json localhtml images
```

Comment:
This adds `flow` to the dependencies for building this project. We can't add any dependencies.

Reply:
Shall we run `make flow` directly as part of CircleCI?

Reply:
Yeah :) *before* the `make -j8` because that sometimes fails due to APy being down (hence the `|| true`.

Reply:
Also, why are `make` errors suppressed?

Reply:
Also, I might open another ticket for typing html-tools with flow if you're interested.

Reply:
Sure, put together the specs.

Reply:
Will look up the syntax and stuff while you are writing it up.